### ISSUE TITLE
[Labels & Instructions – Make Code - Explorer] Visual label and aria-label are not same for 'Explorer' button under 'Editor' screen.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3639]
**User Experience**:
If visual label of some controls is different from the aria label defined for them, the screen reader dependent user will get confused regarding the name of the controls. Thus, the user may get disoriented.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Java Script" drop down and select it.
6. Navigate to "Python" option and select it.
7. Navigate to "Explorer" and select it.
8. Navigate through all the controls after expanding Explorer control.
9. Open developer tool using F12 key and navigate to ‘Explorer' button.
10. Verify the issue.

**Actual Result:**
Visual label and aria-label are not same for 'Explorer' button under 'Editor' screen and screen reader announces 'File explorer toolbar'.

**Expected Result:**
Visual label and aria-label should be same for 'Explorer' button under 'Editor' screen, so that screen reader could read it correctly.

**Note:**
Same issue appear under Chrome & Firefox.

**WCAG Reference:**
[WCAG 2.5.3 - Label in Name
](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, NVDA, JAWS

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG2.5.3, #Win10-Edge(Chromium)

![WCAG2 5 3_Explorer](https://user-images.githubusercontent.com/72871089/98770104-76b6e200-2407-11eb-9fcd-12cfaa20daeb.jpg)
